job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -466,7 +492,13 @@ class WebApi(system: ActorSystem,
                       JobManagerActor.StartJob(appName, classPath, jobConfig, events))(timeout)
                     respondWithMediaType(MediaTypes.`application/json`) { ctx =>
                       future.map {
-                        case JobResult(_, res)       => ctx.complete(resultToTable(res))
+                        case JobResult(_, res) => {
```

Comment:
No need for braces around the case code block.  This style is not used elsewhere in the code.